modules/ROOT/pages/rtf-scale.adoc

```diff
@@ -69,4 +69,6 @@ Aspects to consider:
 * The performance of large-scale deployment orchestration through Runtime Fabric is inversely proportional to the volume of deployments.
 
 * RTFCTL clusters are supported only for certain benchmarks.
-* Using Helm enables you to manage any cluster that requires customized resources for Runtime Fabric agent due to the size of the cluster.
\ No newline at end of file
+* Using Helm enables you to manage any cluster that requires customized resources for Runtime Fabric agent due to the size of the cluster.
+* When scaling up Runtime Fabric agent resource limit, consider increasing the request accordingly.
```

Comment:
Hi, I'm a little confused by "consider increasing the request accordingly." Is this referring to request timeout? Request size? Or something else?